### ISSUE TITLE
debian package 'jitsi_2.10.5550-1_amd64.deb' - improper dependency handling, install fails

### ISSUE THREAD

**Issue by mabra** [jitsi/jitsi#712]
## Description
Installation of the current version for and on debian buster fails.

## Current behavior
Shows error on dependencies and installation fails.
Log:
`
dpkg -i jitsi_2.10.5550-1_amd64.deb 
Selecting previously unselected package jitsi.
(Reading database ... 178516 files and directories currently installed.)
Preparing to unpack jitsi_2.10.5550-1_amd64.deb ...
Unpacking jitsi (2.10.5550-1) ...
dpkg: dependency problems prevent configuration of jitsi:
 jitsi depends on libappindicator1; however:
  Package libappindicator1 is not installed.
 jitsi depends on jitsi-archive-keyring; however:
  Package jitsi-archive-keyring is not installed.

dpkg: error processing package jitsi (--install):
 dependency problems - leaving unconfigured
Processing triggers for gnome-menus (3.31.4-3) ...
Processing triggers for desktop-file-utils (0.23-4) ...
Processing triggers for mime-support (3.62) ...
Processing triggers for man-db (2.8.5-2) ...
Errors were encountered while processing:
 jitsi
`
## Expected Behavior
Installs and runs properly.

## Possible Solution
Unknown.
A probably separate installation of mentioned dependencies would produce other errors in the future.

## Steps to reproduce
Download 'jitsi_2.10.5550-1_amd64.deb' and issue:
"dpkg -i jitsi_2.10.5550-1_amd64.deb

# Environment details
Linux box2 5.8.0-0.bpo.2-amd64 #1 SMP Debian 5.8.10-1~bpo10+1 (2020-09-26) x86_64 GNU/Linux
Debian GNU/Linux 10 (buster)

**Comment by mabra** [comment]
Addendum.
I was not too lazy and went to a "snap-shot-able" machine (lxc container running debian/xfce4) and installed jitsi from the same source - resulting in the same error.
Then I looked for the failed dependencies:
- jitsi-archive-keyring: is not in the OS's repository
- libappindicator1: offered the next problem ....
`
 libappindicator1 : Depends: libdbusmenu-gtk4 (>= 0.4.2) but it is not going to be installed
                    Depends: libindicator7 (>= 0.4.90) but it is not going to be installed
E: Unmet dependencies. Try 'apt --fix-broken install' with no packages (or specify a solution).
`
Just as a try to fix it by myself (in opposit to my comment above - this way ist not a good idea for a future stabil host).

**Comment by tagMacher** [comment]
Is the jitsi archive keyring available as a deb file? That would be a proper way to solve this issue.

**Comment by jcalfee** [comment]
For now, I think this link may belong with the install instructions: http://download.jitsi.org/stable/jitsi-archive-keyring_1.0.1_all.deb

**Comment by zecamigo** [comment]
> For now, I think this link may belong with the install instructions: http://download.jitsi.org/stable/jitsi-archive-keyring_1.0.1_all.deb

Yes, please add installation instructions for debian and ubuntu packages. And please use "httpS".

**Comment by joker-x** [comment]
This script install jitsi desktop on Ubuntu 20.04 and 22.04:

https://gist.github.com/joker-x/55bc16d4f8066e4a4eb9dff3873a423f
